dspace-api/src/test/java/org/dspace/content/RelationshipMetadataServiceTest.java

```diff
@@ -496,6 +497,123 @@ public void testGetNextLeftPlace() throws Exception {
         assertThat(relationshipService.findNextLeftPlaceByLeftItem(context, leftItem), equalTo(2));
 
 
+    }
 
+    @Test
+    public void testGetVirtualMetadata() throws SQLException, AuthorizeException {
```

Comment:
Can we update this test method to use Builder classes?  Currently, it does not appear to be cleaning up after itself.